### DISCUSSION TITLE
Pricing: can I disable metrics?

### DISCUSSION THREAD

**Discussion by nbcchen** [serverless/serverless#12764] [General]
According to https://www.serverless.com/pricing metrics and traces are not free. Is there a way to turn them off?

**Comment by lucaseneasneves** [comment]
Having the same questions, not to not pay for them but due to legal and security reasons we can't have any 3rd-party collecting our data to their servers.

**Comment by garethmcc** [comment]
Hi there folks. With V4 metrics and traces are opt in and require a user to specifically turn them on and activate it withing any AWS account you want the metrics and traces to operate in. These are not required or forced in any way.
